infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -183,6 +213,54 @@ public static void main(String[] args) {
         addManagementServer();
         router.baseServer.setOptionsMap(opts);
 
+        // Setup SSL if needed
+        Boolean tlsEnabled = (Boolean) opts.get("--enable-tls");
+        if (tlsEnabled) {
+            // Get the TLS cipher suites to enable
+            String ciphs = (String) opts.get("--tls-ciphers");
+            if (ciphs != null) {
+                List<String> ciphers = Pattern.compile(",")
+                    .splitAsStream(ciphs)
+                    .map(String::trim)
+                    .collect(Collectors.toList());
+                enabledTlsCipherSuites = ciphers.toArray(new String[ciphers.size()]);
+            }
+
+            // Get the TLS protocols to enable
+            String protos = (String) opts.get("--tls-protocols");
+            if (protos != null) {
```

Comment:
If this option is not set, opts should have the default value of TLSv1.1,TLSv1.2. So protos should not be null.
If it is null at this point for some reason, enabledTlsProtocols will be null and the later call to setEnabledProtocols() will throw an IllegalArgumentException, which is what would happen even if the user sets this option to some junk.